build_runner_core/lib/src/environment/io_environment.dart

```diff
@@ -17,6 +17,8 @@ import '../package_graph/package_graph.dart';
 import 'build_environment.dart';
 import 'create_merged_dir.dart';
 
+final _logger = new Logger('IOEnvironment');
```

Comment:
need to include this at https://github.com/dart-lang/build/blob/a10c67f7ef87bae6e942cd8ccc7dfc1903e5afa1/build_runner/lib/src/logging/std_io_logging.dart#L69

Reply:
done